### PR TITLE
Choose Primary domain from Dashboard

### PR DESCRIPTION
**Pull request by EdwardAngert**
To be merged with AL-2142 (PR 2276)

## Effect
PR includes the following changes:
- adds steps to reflect new choose primary workflow on site dashboards
- removes php snippet option

## Remaining Work
- [ ] Confirm removal  of PHP snippet support with @ari-gold 
- [x] Copy Review
- [x] Screenshot to Step 4 of [Add a Custom Domain](https://pantheon.io/docs/domains#add-a-custom-domain) (at least)

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report
- [ ] Archive from **Done** in [Main Project](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/partials/primary-domain.md [hunk 1]
```diff
@@ -1,8 +1,19 @@
+### Set a Primary Domain via the Dashboard
+
+1. From the environment you want to set a primary domain for (typically Test or Live), navigate to **<span class="glyphicons glyphicons-global"></span> Domains / HTTPS**.
+
+1. Ensure that all domains have been added and are listed.
+
+1. In the **Choose Primary Domain** section, select the domain to which traffic should be redirected, and click **Save Configuration**.
+
+   - You can track the task in **Workflows** <Icon icon="chevron-down" />.
```
Comment (ari-gold) [inline source/partials/primary-domain.md]: Is this line about the workflow needed? Seems like too much info.

Comment (alexfornuto) [reply source/partials/primary-domain.md]: I'm fine with removing it... @EdwardAngert?

Comment (EdwardAngert) [reply source/partials/primary-domain.md]: +1
tracking the status in Workflows is a Platform given. I can remove it

Comment (EdwardAngert) [reply source/partials/primary-domain.md]: resolved in #5210

**Comment by alexfornuto** [comment]
Reassigning to @EdwardAngert to see if he still wants @ari-gold to review the thing he was asked to review in the PR description.

**Comment by EdwardAngert** [comment]
considering this done for now. The PHP snippet, if it isn't already deprecated, will be, and dancing back and forth with adding it back in, only to remove it later, just seems gratuitous.

**Pull request merged**